pkg/volume/csi/csi_attacher.go

```diff
@@ -630,6 +625,11 @@ func verifyAttachmentStatus(attachment *storage.VolumeAttachment, volumeHandle s
 		klog.Error(log("attachment for %v failed: %v", volumeHandle, attachErr.Message))
 		return false, errors.New(attachErr.Message)
 	}
+	// if being deleted, fail fast
```

Comment:
This seems problematic because we'll return attached for something that could be in the process of being detached.

Having a deletion timestamp means that we want the volume to be detached. Can you look at the csi-attacher logs to see why the detach is failing?

Reply:
I cannot find the csi-attacher logs in 2020-11-28 since it's already restarted, point is csi-attacher could not recover and it's stuck there, that VolumeAttachment always has a deletion timestamp. Is there anyway to recover this? This issue already happened multiple times.

I could only find below logs in csi-attacher logs:
```
I1201 03:28:08.554290       1 controller.go:198] Started VA processing "csi-0e4e8d6e0077022980b3de60b3fd46060b4cd8feca96eef7126a4f5e75a65680"
I1201 03:28:08.555342       1 trivial_handler.go:53] Trivial sync[csi-0e4e8d6e0077022980b3de60b3fd46060b4cd8feca96eef7126a4f5e75a65680] started
I1201 03:38:08.554534       1 controller.go:198] Started VA processing "csi-0e4e8d6e0077022980b3de60b3fd46060b4cd8feca96eef7126a4f5e75a65680"
I1201 03:38:08.554543       1 trivial_handler.go:53] Trivial sync[csi-0e4e8d6e0077022980b3de60b3fd46060b4cd8feca96eef7126a4f5e75a65680] started
```

Reply:
It's a bug in csi-attacher if it cannot recover. Is this happening in an e2e test? Would increasing log level of attacher help?